docs/examples.md

```diff
@@ -0,0 +1,232 @@
+        uses: actions/checkout@v3
+      - name: 'Dependency Review'
+        uses: actions/dependency-review-action@v3
+        with:
+          config-file: './.github/dependency-review-config.yml'
+```
+
+## Using a configuration file from a external repository
+
+The following example will use a configuration file from an external public GitHub repository to configure the action.
+
+Let's say that the configuration file is located in `github/octorepo/dependency-review-config.yml@main`
+
+The Dependancy Review Action workflow file will then look like this:
+
+```yaml
+name: 'Dependency Review'
+on: [pull_request]
+
+permissions:
+  contents: read
+
+jobs:
+  dependency-review:
+    runs-on: ubuntu-latest
+    steps:
+      - name: 'Checkout Repository'
+        uses: actions/checkout@v3
+      - name: 'Dependency Review'
+        uses: actions/dependency-review-action@v3
+        with:
+          config-file: 'github/octorepo/dependency-review-config.yml@main'
+```
+
+## Using a configuration file from a external repository with a personal access token
+
+The following example will use a configuration file from an external private GtiHub repository to configure the action.
+
+Let's say that the configuration file is located in `github/octorepo-private/dependency-review-config.yml@main`
+
+The Dependancy Review Action workflow file will then look like this:
+
+```yaml
+name: 'Dependency Review'
+on: [pull_request]
+
+permissions:
+  contents: read
+
+jobs:
+  dependency-review:
+    runs-on: ubuntu-latest
+    steps:
+      - name: 'Checkout Repository'
+        uses: actions/checkout@v3
+      - name: 'Dependency Review'
+        uses: actions/dependency-review-action@v3
+        with:
+          config-file: 'github/octorepo-private/dependency-review-config.yml@main'
+          config-file-token: ${{ secrets.GITHUB_TOKEN }} # or a personal access token
+```
+
+## Getting the results of the action in the PR as a comment
+
+Using the `comment-summary-in-pr` you can get the results of the action in the PR as a comment. In order for this to work, the action needs to be able to create a comment in the PR. This requires additional `pull-requests: write` permission.
+
+```yaml
+name: 'Dependency Review'
+on: [pull_request]
+
+permissions:
+  contents: read
+  pull-requests: write
+
+jobs:
+  dependency-review:
+    runs-on: ubuntu-latest
+    steps:
+      - name: 'Checkout Repository'
+        uses: actions/checkout@v3
+      - name: 'Dependency Review'
+        uses: actions/dependency-review-action@v3
+        with:
+          fail-on-severity: critical
+          deny-licenses: LGPL-2.0, BSD-2-Clause
+          comment-summary-in-pr: true
+```
+
+## Exclude dependencies from the license check
+
+Using the `allow-dependencies-licenses` you can exclude dependencies from the license check. You can exclude dependencies per ecosystem and the values should be provided in a valid [purl](https://github.com/package-url/purl-spec) format.
```

Comment:
I'm not sure the phrase "per ecosystem" makes sense to me here, but maybe I'm missing something. PURLs necessarily refer to an ecosystem already, right?

```suggestion
Using the `allow-dependencies-licenses` you can exclude dependencies from the license check. The values should be provided in [purl](https://github.com/package-url/purl-spec) format.
```